javalin-testtools/src/test/java/io/javalin/testtools/JavaTest.java

```diff
@@ -57,7 +57,7 @@ public void can_do_query_params_and_headers() {
     @Test
     public void post_with_json_serialization_works() {
         JavalinTest.test((server, client) -> {
-            server.post("/hello", ctx -> ctx.result(ctx.bodyAsClass(MyJavaClass.class).field1));
+            server.post("/hello", ctx -> ctx.result(ctx.<MyJavaClass>bodyAsClass(MyJavaClass.class).field1));
```

Comment:
Why is this required?

Reply:
`Type` does not contain generic signature like `Class<T>`, so we need to tell what's the expected type. I guess we could also try overload method with second `Class<?>` signature so this issue should be gone if we'd decide to cleanup & merge this.